### DISCUSSION TITLE
Intéressé pour donner un coup de main !

### DISCUSSION THREAD

**Discussion by kevinnozeno** [uurshin/minotaure#97] [Général / General]
Hello, j'étais présent pour la partie Minotaure au Rol'Event le 16 Octobre et je serais peut-être intéressé pour vous donner un coup de main sur le projet. Pourrions-nous échanger en MP ?

**Comment by uurshin** [comment]
Yes ! Je vois pas de mp github, mais si tu peux rejoindre le discord minotaure (tu trouveras le lien sur le github) je t'ajoute en privé quand je te vois passer.
